app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -113,6 +134,19 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
       }
     }
 
+    /// Final [rows] to be inserted to [BigQuery]
+    final TableDataInsertAllRequest rows =
+      TableDataInsertAllRequest.fromJson(<String, Object>{
+      'rows': tableDataInsertAllRequestRows
+    });
+
+    /// Insert [commits] to [BigQuery]
+    try {
+      await tabledataResourceApi.insertAll(rows, projectId, dataset, table);
+    } catch (error) {
```

Comment:
We should catch specific exceptions we know this can throw - avoid generic `catch`es.

Reply:
Replaced with [ApiRequestError]